source/uwp/Renderer/lib/AdaptiveCardRendererComponent.cpp

```diff
@@ -300,6 +296,31 @@ namespace AdaptiveNamespace
         m_actionSentimentResourceDictionary = actionSentimentResourceDictionary;
     }
 
+    static HRESULT TryInsertResourceToResourceDictionaries(_In_ IResourceDictionary* resourceDictionary,
```

Comment:
>TryInsertResourceToResourceDictionaries [](start = 19, length = 39)

Look like this is only used to update m_actionSentimentResourceDictionary on this class. Consider making this a private class method TryInsertResourceIntoActionSentimentDictionary that doesn't need the resource dictionary passed in.